packages/entando-apimanager/modules/api/apiManager.js

```diff
@@ -146,7 +145,7 @@ export const makeRealRequest = (request, page) => {
   validateRequest(request);
   if (request.useAuthentication && !getAuthenticationToken()) {
     store.dispatch(logoutUser());
-    return new Promise(resolve => resolve({ ok: false, status: 401 }));
+    return Promise.reject(new Error('app.permissionDenied'));
```

Comment:
why this change? @nicpuddu I guess lots of thunks rely on a resolved promise with 401 status

Reply:
thing is reject throws an error, and it should be returned only if the request fails horribly, so yeah I agree with @gidesan

Reply:
okay. I have restored the resolve promise indicated